src/Model/Item.php

```diff
@@ -1364,11 +1411,21 @@ private static function storeForUser(array $item, int $uid)
 		}
 
 		unset($item['id']);
-		unset($item['parent']);
 		unset($item['mention']);
 		unset($item['starred']);
 		unset($item['unseen']);
 		unset($item['psid']);
+		unset($item['pinned']);
+		unset($item['ignored']);
+		unset($item['pubmail']);
+		unset($item['forum_mode']);
+
+		//unset($item['post-reason']);
+		//unset($item['protocol']);
```

Comment:
Please don't add commented out code, instead stash it in git if you need it for later.

Reply:
Sorry, I forgot to push to additional commits I did today.